docs/ref/middleware.txt

```diff
@@ -428,6 +428,9 @@ Here are some hints about the ordering of various Django middleware classes:
 
    After ``UpdateCacheMiddleware``: Modifies ``Vary`` header.
 
+   It must come before any middleware that may raise an exception if you're
```

Comment:
To match the style of other phrases, put it above "After" and chop "It must come".